### PR TITLE
POC how to make the execution of not ready scripts much more easy

### PR DESCRIPTION
**Pull request by szaimen**
As the title says.
@enoch85 what do you think about this? Would this be okay?

### CODE REVIEW

**Comment by enoch85** [comment]
I'm fine with everything except putting the script in the released version: `nextcloud-startup-script.sh`

File: nextcloud-startup-script.sh [hunk 1]
```diff
@@ -217,6 +217,7 @@ download_script MENU menu
 download_script MENU server_configuration
 download_script MENU nextcloud_configuration
 download_script MENU additional_apps
+download_script NOT_SUPPORTED not-supported
```
Comment (szaimen) [inline nextcloud-startup-script.sh:220]: > I'm fine with everything except putting the script in the released version: `nextcloud-startup-script.sh`

Do you refer to this line?

Comment (enoch85) [reply nextcloud-startup-script.sh:220]: Yes.

Comment (szaimen) [reply nextcloud-startup-script.sh]: okay, I've removed it from the startup script

File: nextcloud_install_production.sh [hunk 2]
```diff
@@ -824,6 +824,7 @@ download_script GITHUB_REPO nextcloud-startup-script
 download_script STATIC instruction
 download_script STATIC history
 download_script NETWORK static_ip
+download_script NOT_SUPPORTED not-supported
```
Comment (szaimen) [inline nextcloud_install_production.sh:827]: would it be okay in the install_production script?

Comment (enoch85) [reply nextcloud_install_production.sh:827]: hehe, no, same thing. :)

Comment (szaimen) [reply nextcloud_install_production.sh:827]: In this case, I guess this PR doesn't make much sense at all

**Comment by szaimen** [comment]
Closing this to keep the not-supported folder clean.

**Pull request closed without merging**